Cargo.toml

```diff
@@ -171,8 +171,8 @@ tl = { git = "https://github.com/astral-sh/tl.git", rev = "6e25b2ee2513d75385101
 tokio = { version = "1.40.0", features = ["fs", "io-util", "macros", "process", "rt", "signal", "sync"] }
 tokio-stream = { version = "0.1.16" }
 tokio-util = { version = "0.7.12", features = ["compat", "io"] }
-toml = { version = "0.8.19" }
-toml_edit = { version = "0.22.21", features = ["serde"] }
+toml = { version = "0.9.2", features = ["fast_hash"] }
```

Comment:
This also transitively enables the `preserve_order`, which [is apparently faster](https://epage.github.io/blog/2025/07/toml-09/), not really sure why it's not enabled by default.